cdc/model/mounter.go

```diff
@@ -17,9 +17,21 @@ import (
 	"math"
 
 	"github.com/pingcap/log"
+	"github.com/pingcap/tidb/types"
 	"go.uber.org/zap"
 )
 
+// RowChangedDatums is used to store the changed datums of a row.
+type RowChangedDatums struct {
```

Comment:
How about define it in cdc/entry/mounter.go?

Reply:
It will cause cycle import.